src/utils.sh

```diff
@@ -20,5 +22,25 @@ function kworkflow-help()
     "\tup,u - Wake up vm\n" \
     "\tcodestyle - Apply checkpatch on directory or file\n" \
     "\tmaintainers - Return the maintainers and the mailing list\n" \
+    "\tsearch - Search for expression on git log or directory\n" \
     "\thelp"
 }
+
+function search()
+{
+  if [[ "$#" -eq 0 ]]; then
```

Comment:
You want to be more strict about this, note that I can pass thousands of parameter with this validation. Additionally, add a unit test to check the parameters values.

Reply:
> You want to be more strict about this, note that I can pass thousands of parameter with this validation. 

As of this new patch, this is intended behavior. `kw explore` now accepts multiple files/directories when searching for expressions.

> Additionally, add a unit test to check the parameters values.

I don't see how this can be done, as parameters are either directories/files, or just a regular expression. The way I see it, any value can be passed no problem.